website/docs/audit.md

```diff
@@ -111,7 +111,8 @@ All of these events (including `violation_audited`) are marked with the same `au
 
 - Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`)
 - Audit chunk size: set `--audit-chunk-size=500` (defaults to `0` = infinite) to limit memory consumption of the auditing `Pod`
-- Audit interval: set `--audit-interval=123` (defaults to every `60` seconds). Disable audit interval by setting `--audit-interval=0`
+- Audit chunk size: set `--audit-chunk-size=400` (defaults to `500`, `0` = infinite) Lower chunk size can reduce memory consumption of the auditing `Pod` but can increase the number requests to the Kubernetes API server.
```

Comment:
this is defined in previous bullet. do you want to modify that with new behavior?

Reply:
looks like you removed the wrong line